frontend/src/pages/Admin/Components/JoinUs/joinus.module.scss

```diff
@@ -0,0 +1,10 @@
+.card-container {
```

Comment:
Please format this file. Each class should be separated by a single blank line and the code should be formatted